index.html

```diff
@@ -0,0 +1,207 @@
+              <ul class="built-with">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Jav Script</li>
+                <li>html</li>
+              </ul>
+              <input type="button" class="work-button" value="See Project" />
+            </div>
+          </div>
+          <div class="my-work-card">
+            <div class="card-image"></div>
+            <div class="my-works-text">
+              <h2>Multi-post Stories Gain+Glory</h2>
+              <ul class="built-with">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Jav Script</li>
+                <li>html</li>
+              </ul>
+              <input type="button" class="work-button" value="See Project" />
+            </div>
+          </div>
+          <div class="my-work-card">
+            <div class="card-image"></div>
+            <div class="my-works-text">
+              <h2>Multi-post Stories Gain+Glory</h2>
+              <ul class="built-with">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Jav Script</li>
+                <li>html</li>
+              </ul>
+              <input type="button" class="work-button" value="See Project" />
+            </div>
+          </div>
+          <div class="my-work-card">
+            <div class="card-image"></div>
+            <div class="my-works-text">
+              <h2>Multi-post Stories Gain+Glory</h2>
+              <ul class="built-with">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Jav Script</li>
+                <li>html</li>
+              </ul>
+              <input type="button" class="work-button" value="See Project" />
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about-me-section">
+        <div class="about-me">
+          <h2>About me</h2>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <input type="button" class="about-me-button" value="Get My Resume" />
+        </div>
+        <div class="grid-container2">
+          <div class="my-work-card2">
+            <div class="card-image2">
+              <img src="image/Ellipse.png" alt="eclipse icon" />
+            </div>
+            <div class="my-works-text2">
+              <h2>Languages</h2>
+              <ul class="built-with2">
+                <li>JavaScript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+              </ul>
+            </div>
+          </div>
+          <div class="my-work-card2">
+            <div class="card-image2">
+              <img src="image/multiple-box.png" alt="eclipse icon" />
+            </div>
+            <div class="my-works-text2">
```

Comment:
- [ ] Kindly make sure all your images are displaying correctly in the "about me section" 🙏 and you use the correct colours as seen in the Figma template 👍  .

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/177016103-4899da51-25e9-432c-875c-7acaedc22889.png)  |  ![image](https://user-images.githubusercontent.com/74814780/177016100-10ebdb35-2a83-47fe-86e8-d231a7f73e9c.png)